lib/chitter_homepage.rb

```diff
@@ -0,0 +1,26 @@
+require 'pg'
+
+class ChitterPage
+  def self.view
+    if ENV["ENVIRONMENT"] == "test"
+      connection = PG.connect(dbname: "chitter_test")
+    else
+      connection = PG.connect(dbname: "chitter")
+    end
+    total = connection.exec("SELECT * FROM peeps ORDER BY id DESC;")
+
+    total.map do |peep|
+      { message: peep["message"], date_posted: peep["date_posted"] }
+    end
+  end
+
+  def self.create(message)
+    if ENV["ENVIRONMENT"] == "test"
+      connection = PG.connect(dbname: "chitter_test")
+    else
+      connection = PG.connect(dbname: "chitter")
+    end 
+
+    connection.exec("INSERT INTO peeps (message) VALUES('#{message}')")
```

Comment:
In order to protect the app from injection, you can use `.exec_params` method instead of `.exec`.
And you can pass the message with this syntax:
`connection.exec_params("INSERT INTO peeps (message) VALUES($1);", [message])`